<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-09-11T03:53:25 IST" name="Default suite" finished-at="2025-09-11T03:53:39 IST" duration-ms="13056">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.Run@ed7f8b4]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2025-09-11T03:53:25 IST" name="Default test" finished-at="2025-09-11T03:53:39 IST" duration-ms="13056">
      <class name="TestRunner.Run">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:TestRunner.Run@ed7f8b4]" started-at="2025-09-11T03:53:26 IST" name="setUpClass" finished-at="2025-09-11T03:53:26 IST" duration-ms="869" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@7d904ff1]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.Run@ed7f8b4]" started-at="2025-09-11T03:53:26 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2025-09-11T03:53:38 IST" duration-ms="12057" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[""]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Customer"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at StepDefination.StepDef.user_can_view_confirmation_message(StepDef.java:162)
at âœ½.User can view confirmation message "The new customer has been added successfully."(file:///C:/Users/KIIT/eclipse-workspace-Practice/AutomationProject/CucumberFramework/./Features/Customer.feature:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:TestRunner.Run@ed7f8b4]" started-at="2025-09-11T03:53:38 IST" name="tearDownClass" finished-at="2025-09-11T03:53:39 IST" duration-ms="65" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- TestRunner.Run -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
